### ISSUE TITLE
Consolidate 

### ISSUE THREAD

**Issue by fergie-nz** [msupply-foundation/open-msupply#4046]
Consolidate NZ(British) vs US spelling across app.

Should our english app be in US or British english? Should we try have consistency across OMS?
Should we try to have consistency across all of our apps and docs?

## Describe the solution you'd like 🎁

- Decide on a language
- Change all spelling to be consistent with that language

**Comment by adamdewey** [comment]
Nice Ferg, I think possibly best solution would to be have a translation for each and have one as default?

**Comment by mark-prins** [comment]
we are using British / NZ English yup.
The issue arose with the new immunisation work - given that the client for that is using US English, all of the reference docs refer to `Immunization` and `Catalog`. I'm somewhat reluctantly leaning to US spellings for those, even though I don't like 'catalog'

**Comment by roxy-dao** [comment]
> we are using British / NZ English yup. The issue arose with the new immunisation work - given that the client for that is using US English, all of the reference docs refer to `Immunization` and `Catalog`. I'm somewhat reluctantly leaning to US spellings for those, even though I don't like 'catalog'

Can we just have two English lang??

**Comment by mark-prins** [comment]
yes, can do. Similar to how we have two Français - adding in an `en-US` option would be fine. and better than a complete change. the only decision then is whether to default to US or BR?
Thanks @roxy-dao 🙏

**Comment by roxy-dao** [comment]
> yes, can do. Similar to how we have two Français - adding in an `en-US` option would be fine. and better than a complete change. the only decision then is whether to default to US or BR? Thanks @roxy-dao 🙏

I prefer British, but guessing en-US is more popular because of economic and cultural influences >.>
